scene/gui/label.cpp

```diff
@@ -609,6 +609,7 @@ void Label::_notification(int p_what) {
 
 		case NOTIFICATION_RESIZED: {
 			lines_dirty = true;
+			_shape();
```

Comment:
Probably should be `queue_redraw();` instead, processing text should not be necessary until control is displayed.